.github/workflows/test_deploy.yml

```diff
@@ -147,7 +149,9 @@ jobs:
         git config --global --add user.email "renku@datascience.ch"
     - name: Test with pytest
       env:
+        DATAVERSE_ACCESS_TOKEN: ${{ secrets.DATAVERSE_ACCESS_TOKEN }}
         IT_OAUTH_GIT_TOKEN: ${{ secrets.IT_OAUTH_GIT_TOKEN }}
+        ZENODO_ACCESS_TOKEN: ${{ secrets.ZENODO_ACCESS_TOKEN }}
       run: pytest -m integration -v
 
   # test-macos-integration-pr:
```

Comment:
What's the reason to keep this commented out? Should we just delete it? Imho, It's part of git history anyway so no need to keep dead code around.

Reply:
Agreed! Please approve!